biojava-core/src/main/java/org/biojava/nbio/core/sequence/io/embl/EmblId.java

```diff
@@ -0,0 +1,77 @@
+package org.biojava.nbio.core.sequence.io.embl;
+
+/**
+ * This class should parse the data of embl file identification
+ * @author Noor Aldeen Al Mbaidin
```

Comment:
Could you add `@since 5.0.0` tag to all the new classes here? Thanks!

Reply:
sure.